.gitlab-ci.yml

```diff
@@ -0,0 +1,70 @@
+include:
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-github-status-updates.yml'
+
+stages:
+  - test
+  - publish
+
+test:unit:
+  image: golang:1.14
+  stage: test
+
+  before_script:
+    # Rename the branch we're on, so that it's not in the way for the
+    # subsequent fetch. It's ok if this fails, it just means we're not on any
+    # branch.
+    - git branch -m temp-branch || true
+    # Git trick: Fetch directly into our local branches instead of remote
+    # branches.
+    - git fetch origin 'refs/heads/*:refs/heads/*'
+    # Get last remaining tags, if any.
+    - git fetch --tags origin
+
+    # Prepare GO path
+    - mkdir -p /go/src/github.com/mendersoftware
+    - cp -r $CI_PROJECT_DIR /go/src/github.com/mendersoftware/openssl
+    - cd /go/src/github.com/mendersoftware/openssl
+
+    # Install code coverage / coveralls tooling
+    - go get -u github.com/axw/gocov/gocov
+    - go get -u golang.org/x/tools/cmd/cover
+
+    # Install OpenSSL
+    - apt-get update && apt-get install -yyq liblzma-dev libssl-dev
+
+  script:
+    # Test if code was formatted with 'go fmt'
+    # Command will format code and return modified files
+    # fail if any have been modified.
+    - if [ -n "$(go fmt)" ]; then echo 'Code is not formatted with "go fmt"'; false; fi
+
+    # Perform static code analysys
+    - go vet `go list ./... | grep -v vendor`
```

Comment:
I'm pretty sure the inverse grep is not needed. `go list ./...` should ignore vendor by default.